app/javascript/controllers/admin/user_controller.js

```diff
@@ -0,0 +1,25 @@
+import { Controller } from 'stimulus'
+
+export default class extends Controller {
+  static targets = ['deletedbutton', 'restorebutton', 'deletedvalue']
+
+  initialize() {
+    if (this.deletedvalueTarget.innerHTML == "true") {
+      this.showRestoreButtonTarget()
+    } else {
+      this.showDeleteButtonTarget()
+    }
+  }
+
+  showDeleteButtonTarget() {
+    this.restorebuttonTarget.style.display = 'none'
+    this.deletedbuttonTarget.style.display = 'block'
+    this.deletedvalueTarget.parentNode.style.color = "black"
+  }
+
+  showRestoreButtonTarget(){
+    this.restorebuttonTarget.style.display = 'block'
+    this.deletedbuttonTarget.style.display = 'none'
+    this.deletedvalueTarget.parentNode.style.color = "red"
```

Comment:
I believe technically it would be better to add and remove a class name from the elements so the styling and transition can be handled through CSS.

Reply:
Agree, my intention here was to spell out any CSS needs first, then move it into .scss